### PR TITLE
Do not display the stack trace for SkippingException

### LINKED ISSUE #1298: Stacktrace should not be rendered for SkippingException
Throwing a SkippingException allows to skip a spec example (conditionally). Currently, the stacktrace is rendered in the output like for any error, making such skipping very verbose. This is totally unnecessary IMO (and there is no reason to debug a SkippingException).
See https://travis-ci.org/phpspec/prophecy/jobs/658670644#L1272 for an example.

### PR DESCRIPTION
**Pull request by tentacode**
:wave: hello there ! I've chose a quick-pick to start my Hacktoberfest contributions, and chose phpspec because I've been using for years and never contributed... It's my first, please be gentle :pray:  

closes #1298

### CODE REVIEW

**Comment by tentacode** [comment]
eh. By the way I did not know how this work but it looks like phpspec is not eligible for Hacktoberfest :sob:

```
PRs count if:
Submitted during the month of October AND
Submitted in a public repo AND (
  The PR is labelled as hacktoberfest-accepted by a maintainer OR
  Submitted in a repo with the hacktoberfest topic AND (
    The PR is merged OR
    The PR has been approved
  )
) 
```

see [An update on efforts to reduce spam with Hacktoberfest: introducing maintainer opt-in and more.](https://hacktoberfest.digitalocean.com/hacktoberfest-update)

File: features/runner/developer_skips_specs.feature [hunk 1]
```diff
@@ -36,6 +36,10 @@ Feature: Developer skips examples
       }
 
       """
-    When I run phpspec using the "dot" format
+    When I run phpspec with the "verbose" option
```
Comment (ciaranmcnulty) [inline features/runner/developer_skips_specs.feature:39]: Can you please split this into a new scenario specifically about the verbose mode behaviour (that doesn't need to have 'the suite will pass' as a step)

**Review by ciaranmcnulty — CHANGES_REQUESTED** [review]

File: features/runner/developer_skips_specs.feature (hunk 1 above)
Comment (tentacode) [reply features/runner/developer_skips_specs.feature:77]: makes sense, I added a whole new scenario :+1:

**Comment by ciaranmcnulty** [comment]
Thanks!

**Pull request merged**